polyfill/lib/duration.mjs

```diff
@@ -123,9 +123,11 @@ export class Duration {
     if (!dateParts.length && !timeParts.length) return 'PT0S';
     return `P${dateParts.join('')}${timeParts.join('')}`;
   }
-  toLocaleString(...args) {
+  toLocaleString() {
```

Comment:
I would maybe make this forward-compatible by doing something like:
```js
toLocaleString(...args) {
  if (!ES.IsDuration(this)) throw new TypeError('invalid receiver');
  if (typeof Intl.DurationFormat !== 'undefined')
    return new Intl.DurationFormat(...args).format(this);
  console.warn('Temporal.Duration.toLocaleString() requires Intl.DurationFormat.');
  return ES.DurationToString(this);
}
```
I'm not even sure we should have the warning since presumably the spec text will just say to defer to toString() in case of no Intl.DurationFormat.

Reply:
`typeof Intl !== 'undefined' && typeof Intl.DurationFormat !== 'undefined'`